subsys/net/ip/ipv6.h

```diff
@@ -134,7 +134,8 @@ int net_ipv6_send_na(struct net_if *iface, const struct in6_addr *src,
  * @param iface Network interface
  * @param next_header_proto Protocol type of the next header after IPv6 header.
  *
- * @return Return network packet that contains the IPv6 packet.
+ * @return Return network packet that contains the IPv6 packet or NULL if
```

Comment:
If you edit this description, maybe makes sense to make it less cryptic at the same time. If this function returns its input "pkt" param, makes sense to say so, and then in parens add "(which now contains IPv6 header)".